jdi-light/src/main/java/com/epam/jdi/light/settings/CommonSettings.java

```diff
@@ -1,14 +1,33 @@
 package com.epam.jdi.light.settings;
 
-import static com.epam.jdi.light.settings.Strategies.*;
-import static com.epam.jdi.tools.PathUtils.*;
+import static com.epam.jdi.light.settings.Strategies.JDI;
+import static com.epam.jdi.tools.PathUtils.mergePath;
+import static com.epam.jdi.tools.PathUtils.path;
 
 public class CommonSettings {
-    public Strategies strategy = JDI;
-    public String testPropertiesPath = "test.properties";
-    public String logsPath = mergePath("target", ".logs");
-    public String killBrowser = "afterAndBefore";
+    private static CommonSettings commonSettings;
+
+    private CommonSettings() {
+        strategy = JDI;
+        testPropertiesPath = "test.properties";
+        logsPath = mergePath("target", ".logs");
+        killBrowser = "afterAndBefore";
+        projectPath = path("");
+        testPath = mergePath(projectPath, "src", "test");
+    }
+
+    public static CommonSettings getCommonSettings() {
+        if (commonSettings == null) {
+            commonSettings = new CommonSettings();
+        }
+        return commonSettings;
+    }
+
+    public Strategies strategy;
+    public String testPropertiesPath;
+    public String logsPath;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)